tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -345,6 +356,21 @@ public void testParameter(String type) {
         testBookingIdMethods(vr);
         testReviewIdMethods(vr);
         testUserLoginMethods(vr);
+        testParameterWithObjectAndStyle(vr);
+    }
+
+    private void testParameterWithObjectAndStyle(ValidatableResponse vr) {
+        String headParameters = "paths.'/zepplins/{id}'.head.parameters";
+        String getParameters = "paths.'/zepplins/{id}'.get.parameters";
+
+        vr.body(headParameters, hasSize(1));
+        vr.body(getParameters, hasSize(1));
```

Comment:
Hmm, this result is a bit odd. The method also has an `id` path parameter. If this is passing, I would guess that adding `@Parameter` to the method has overriden all the parameters which would have been discovered from the Jakarta REST annotations on the method.

I'm not sure whether the spec requires it to work like that though.

Reply:
With the `@Path` removed, this now makes sense.